README.md

```diff
@@ -310,6 +325,54 @@ function assignId (req, res, next) {
 }
 ```
 
+#### arguments to custom token formats
+
+Example of a custom token format that uses an argument. It will output the
```

Comment:
@dougwilson how about this example which outputs a safe subset of process information? 

> Example of a custom token format that uses an argument. It will output the
property from `process` related to the argument passed to the token. The valid
arguments are restricted to safe properties of the `process` object in Node.js.

```js
var express = require('express')
var morgan = require('morgan')

morgan.token('process', function getId (req, res, arg) {
  var accessors = {
    arch: function () { return process.arch },
    pid: function () { return process.pid },
    platform: function () { return process.platform },
    title: function () { return process.title },
    uptime: function () { return process.uptime() },
    version: function () { return process.version }
  }

  if (accessors[arg]) {
    return String(accessors[arg]());
  }

  return String('Unknown arg: ' + arg);
})

var app = express()

app.use(morgan(':process[pid] :process[uptime] :response-time'))

app.get('/', function (req, res) {
  res.send('hello, world!')
})
```